libraries/BLE/src/BLERemoteCharacteristic.cpp

```diff
@@ -520,7 +520,6 @@ void BLERemoteCharacteristic::registerForNotify(notify_callback notifyCallback,
 void BLERemoteCharacteristic::removeDescriptors() {
 	// Iterate through all the descriptors releasing their storage and erasing them from the map.
 	for (auto &myPair : m_descriptorMap) {
-	   m_descriptorMap.erase(myPair.first);
 	   delete myPair.second;
 	}
 	m_descriptorMap.clear();   // Technically not neeeded, but just to be sure.
```

Comment:
Just remove this comment to avoid the clear being deleted in the future.

```suggestion
	m_descriptorMap.clear();
```

Reply:
@chesterkwan Can you please commit this suggested change, so we can merge it?